includes/database/DatabasePDO.php

```diff
@@ -23,7 +23,7 @@ class DatabasePDO extends Database
         'FETCH_BOTH' => PDO::FETCH_BOTH,
         'FETCH_NUM' => PDO::FETCH_NUM,
     ];
-    protected $error_supress = !WeBidDebug;
+    protected $error_supress = WeBidDebug;
```

Comment:
The problem with this is it now changes the definitions of WeBidDebug so they no long make sense
WeBidDebug = true will disable debug mode

Think the best solution is to move the definition into the class creator